SharpBgfx/Enums.cs

```diff
@@ -618,7 +618,12 @@ public enum TextureFormat {
         /// <summary>
         /// 8-bit stencil.
         /// </summary>
-        D0S8
+        D0S8,
+
+        /// <summary>
+        /// 
```

Comment:
Can you put a brief summary in here? Something like "The count of all texture formats. Not a valid value."